app/imports/ui/components/SmartContractItem.jsx

```diff
@@ -10,11 +10,16 @@ class SmartContractItem extends React.Component {
       <Table.Row>
         <Table.Cell>{this.props.smartContract.unitAddress}</Table.Cell>
         <Table.Cell>{this.props.smartContract.homeownerName}</Table.Cell>
-        <Table.Cell>{this.props.smartContract.tenetName}</Table.Cell>
-        <Table.Cell>{this.props.smartContract.tenetStance}</Table.Cell>
+        <Table.Cell>{this.props.smartContract.tenantName}</Table.Cell>
+        <Table.Cell>{this.props.smartContract.tenantStance}</Table.Cell>
+        <Table.Cell>{this.props.smartContract.tenantSignature}</Table.Cell>
+        <Table.Cell>{this.props.smartContract.homeownerSignature}</Table.Cell>
         <Table.Cell>
```

Comment:
Optional suggestion for future:

I'd suggest to make this cell hidden for tenants:
```
{ isHomeowner && 
<Table.Cell> ... </Table.Cell> 
```

same for the header row. 

You could pass isHomeowner as a boolean prop.